### PR TITLE
replace HTML tags with Markdown

### PR DESCRIPTION
**Pull request by MatthewKwong**
This pull request replaces the use of HTML tags with Markdown language.

This is most notably seen in:
1. Lists
2. Directly displaying HTML tags on the page

### CODE REVIEW

File: workshops/cringe_101/README.md [hunk 1]
```diff
@@ -374,35 +374,29 @@ simply by typing them in between the `<>`
...
-  <p><code>src="" :</code>
-    <ul>
-      <li>e.g preload="none" would tell it to not load the file when the page loads
-src="".</li>
-      <li>	Specifies where the audio file is. (This won't work on every browser so that's why we use the <source> tags)</li>
-  </ul>
+`autoplay:` Tells the browser to automatically play the audio once it's loaded.
+`controls:` Creates a play bar with a play/pause button, sliding bar, and a volume control
+`loop:` Loops the audio again once it's finished.
+`muted:`Mutes the audio output. (doesn't stop the audio from playing, it just has no volume).
```
Comment (zachlatta) [inline workshops/cringe_101/README.md]: These should be in bullet points.

**Comment by zachlatta** [comment]
Good work so far! Can you make sure that all of your Markdown conforms to our style guide?  https://github.com/hackclub/meta/blob/master/styleguides/markdown.md

File: workshops/cringe_101/README.md [hunk 2]
```diff
@@ -374,35 +374,29 @@ simply by typing them in between the `<>`
...
+    - Specifies where the audio file is. (This won't work on every browser so that's why we use the <source> tags)
+  
+
 Example of an audio tag set to automatically play, and loop an audio file:
 
-```html
-<audio autoplay loop src="hotline-bling.mp3"></audio>
-```
+
+`<audio autoplay loop src="hotline-bling.mp3"></audio>`
```
Comment (zachlatta) [inline workshops/cringe_101/README.md]: Why'd you make this change?

Comment (MatthewKwong) [inline workshops/cringe_101/README.md]: I thought it was incorrect to do 
```html

Comment (zachlatta) [inline workshops/cringe_101/README.md]: Hmm?

File: workshops/cringe_101/README.md [hunk 3]
```diff
@@ -414,39 +406,34 @@ this tutorial we'll just be using it for audio.
 
 This tag also has its own set of attributes like `<audio>` also it's
 **self-closing**.
-<p><code>src="[URL]":</code> Points to the location of the media file
-        - File can be hosted on another website
-        -must end in a supported file extension
+`src="[URL]":`
+- Points to the location of the media file
+    - File can be hosted on another website
+    - must end in a supported file extension
```
Comment (zachlatta) [inline workshops/cringe_101/README.md]: Should only be indented by 2 spaces. Not 4.

File: workshops/cringe_101/README.md [hunk 4]
```diff
@@ -414,39 +406,34 @@ this tutorial we'll just be using it for audio.
...
-
-<p><code>type="[media-type]"</code>
-  <li> Specifies the type of media that will be displayed
-      -e.g <code>;source src="blinghotline.mp3" type="audio/mpeg"&gt;</code>
-</li>
-<br>
- <li>      This changes depending on the type of file you use. You can find more
+- Alternatively, hosted directly where the website is
+
+- e.g "thiswebsite.com/hotlinebling.mp3"
```
Comment (zachlatta) [inline workshops/cringe_101/README.md]: No need for the extra line on 415.

File: workshops/cringe_101/README.md [hunk 5]
```diff
@@ -414,39 +406,34 @@ this tutorial we'll just be using it for audio.
...
+  another example:
+
+![](https://i.imgur.com/LUcCvwl.png)
+
+
+`type="[media-type]"`
```
Comment (zachlatta) [inline workshops/cringe_101/README.md]: No need for the extra newline on 421.

**Comment by MatthewKwong** [comment]
Fixed errors previously called out by Zach.

File: workshops/cringe_101/README.md [hunk 6]
```diff
@@ -374,34 +374,26 @@ simply by typing them in between the `<>`
...
+    - `auto:` load the audio file after the page loads.
+    - `metadata:` only load the metadata (extra info like author, etc that is stored within the file) when the page loads.
+    - `none:` doesn't load the audio file when the page loads.
+ 
+- `src="" :`
+    - e.g preload="none" would tell it to not load the file when the page loads
+- `src=""`
+    - Specifies where the audio file is. (This won't work on every browser so that's why we use the <source> tags)
   
+
```
Comment (zachlatta) [inline workshops/cringe_101/README.md]: Did you intend for there to be an extra line here?

File: workshops/cringe_101/README.md [hunk 7]
```diff
@@ -374,34 +374,26 @@ simply by typing them in between the `<>`
...
 Example of an audio tag set to automatically play, and loop an audio file:
 
 ```html
-<audio autoplay loop src="hotline-bling.mp3"></audio>
+<audio autoplay loop src="hotline-bling.mp3"></audio>`
```
Comment (zachlatta) [inline workshops/cringe_101/README.md]: Is that extra backtick intentional?

**Comment by zachlatta** [comment]
I've made a ton of changes to Matthew's original changes in the updated commit. See the commit description for details.

**Pull request merged**